cockroachcloud/ccloud-get-started.md

```diff
@@ -0,0 +1,434 @@
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster create dedicated dim-dog us-central1:1 --cloud GCP --vcpus 2 --storage-gib 15
+~~~
+
+~~~
+∙∙∙ Creating cluster
+Success! Created cluster
+  name: dim-dog
+  id: ec5e50eb-67dd-4d25-93b0-91ee7ece778d
+~~~
+
+The `id` in the output is the cluster ID. The `name` is used in other `ccloud` commands to identify the cluster on which the `ccloud` command operates.
+
+When creating multi node clusters, you must specify how many nodes should be in each region supported by the cloud infrastructure provider. For example, the following command creates a 3 node cluster where 2 nodes are in `us-central1` and 1 node is in `us-west2`:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster create dedicated dim-dog us-central1:2 us-west2:1 --cloud GCP --vcpus 2 --storage-gib 15
+~~~
+</section>
+
+{{site.data.alerts.callout_info}}
+If you set a maximum spend limit greater than $0 on a {{ site.data.products.serverless }} cluster, or create a {{ site.data.products.dedicated }} cluster, you must [add a credit card](billing-management.html) to your organization.
+{{site.data.alerts.end}}
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+## Create and manage IP allowlists using `ccloud cluster networking allowlist`
+
+Use the `ccloud cluster networking allowlist create` command to create an [IP allowlist](network-authorization.html#ip-allowlisting), which allows incoming network connections from the specified network IP range. Use the `--sql` flag to allow incoming CockroachDB SQL shell connections from the specified network. Use the `--ui` flag to allow access to the DB Console from the specified network.
+
+The IP range must be in [Classless Inter-Domain Routing (CIDR) format](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing). For more information on CIDR, see Digital Ocean's [Understanding IP Addresses, Subnets, and CIDR Notation for Networking](https://www.digitalocean.com/community/tutorials/understanding-ip-addresses-subnets-and-cidr-notation-for-networking#cidr-notation).
+
+For example, to allow incoming connections from a single IP address, 1.1.1.1, to your cluster, including the CockroachDB SQL shell and DB Console, use the following command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster networking allowlist create dim-dog 1.1.1.1/32 --sql --ui
+~~~
+
+~~~
+∙∙∙ Creating IP allowlist entry...
+Success! Created IP allowlist entry for
+ network: 1.1.1.1/32
+ cluster: 041d4c6b-69b9-4121-9c5a-8dd6ffd6b73d
+~~~
+
+Use the `ccloud cluster networking allowlist list` command to list the IP allowlists for your cluster:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster networking allowlist list dim-dog
+~~~
+
+~~~
+∙●∙ Retrieving cluster allowlist...
```

Comment:
Weird ellipsis here too.

Reply:
They're part of the output of the command. When you run it, they animate to show status.